rtmbot.py

```diff
@@ -1,11 +1,9 @@
 #!/usr/bin/env python
 
-import sys
```

Comment:
This sys import is needed here because the next line uses sys. If you add this back and remove the following sys import, I'll try to get this PR merged.